bindings/java/src/org/sleuthkit/datamodel/TimelineManager.java

```diff
@@ -25,6 +25,7 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
+import java.time.Instant;
```

Comment:
Update the year in Copyright statement in file header to 2020 (and in all affected files)

Reply:
FIxed